rt/rs/security/oauth-parent/oauth2-saml/src/main/java/org/apache/cxf/rs/security/oauth2/saml/SamlOAuthValidator.java

```diff
@@ -144,7 +145,7 @@ private void validateSubjectConfirmation(Message m,
                                              SubjectConfirmationData subjectConfData) {
         if (subjectConfData == null) {
             if (!subjectConfirmationDataRequired
-                && cs.getNotOnOrAfter() != null && !cs.getNotOnOrAfter().isBeforeNow()) {
+                && cs.getNotOnOrAfter() != null && !cs.getNotOnOrAfter().isBefore(Instant.now())) {
```

Comment:
should there be an explicit timezone be specified instead of using the JVM default?

Reply:
@coheigea could you please address / clarify ^^ , thank you.

Reply:
@andrei-ivanov The previous code used https://www.joda.org/joda-time/apidocs/org/joda/time/DateTime.html#DateTime-- which uses the default time zone.

Reply:
Well, maybe that was incorrect too, that's why I was hoping someone who knows better can verify

Reply:
@andrei-ivanov Instant.now() uses the System UTC clock, so I think it's correct - https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#now--

Reply:
Hmm, indeed... somehow I confused it with other `now()` calls with use the JVM default.
Sorry for the noise

Reply:
Not at all, I'm happy to have someone review it.